pylint/checkers/format.py

```diff
@@ -418,12 +418,12 @@ def _check_keyword_parentheses(
                         self._check_keyword_parentheses(tokens[i:], 0)
                     return
 
-    def _prepare_token_dispatcher(self):
-        dispatch = {}
-        for tokens, handler in ((_KEYWORD_TOKENS, self._check_keyword_parentheses),):
-            for token in tokens:
-                dispatch[token] = handler
-        return dispatch
+    def _prepare_token_dispatcher(self) -> dict[str, Any]:
```

Comment:
I'd say let's leave the typing out if we don't really add anything. Easier to see that we still need to do this that way.

Reply:
I agree, but we're adding the dict with key that are strings information though ?

Reply:
Hmm, yeah I guess. Can't we solve the `Any`?

Reply:
I don't understand what ``self._check_keyword_parentheses`` returns. There's no return, so I'm under the impression handler is always None, but I tried refactoring this and it started to fail.

Reply:
Wait, it's ``self._check_keyword_parentheses`` not the result of the call.

Reply:
I added the typing and also refactored the comprehension so it's less convoluted in https://github.com/PyCQA/pylint/pull/6452/commits/8882806e46a92c3930e8f3106ad47b4e9522a177

Reply:
I removed the function, the attribute it was creating and the typing entirely. Taking the time to really understand the code is worth it sometime...